furusiya/function_mess.py

```diff
@@ -223,18 +222,13 @@ def play_game():
 
     player_action = None
     Game.mouse_coord = (0, 0)
+    Game.renderer = MapRenderer(Game.area_map, Game.player, Game.ui)
     Game.renderer.recompute_fov = True
-    Game.ui.con.clear()  # unexplored areas start black (which is the default background color)
 
     while not tdl.event.is_window_closed():
-
         # draw all objects in the list
         Game.renderer.render()
 
-        # erase all objects at their old locations, before they move
-        for obj in Game.area_map.entities:
-            obj.clear()
```

Comment:
Why isn't this necessary any more?

Reply:
See map_renderer.py:
https://github.com/nightblade9/furusiya/blob/342ec2e51eedbd70fcd58672b04d1ea6729b39da/furusiya/view/map_renderer.py#L53-L61

Old monster locations are overridden if they're in the FOV. If they're no longer in it:
https://github.com/nightblade9/furusiya/blob/342ec2e51eedbd70fcd58672b04d1ea6729b39da/furusiya/view/map_renderer.py#L30-L36
They are rendered as "explored".

So yes, unneeded.